.github/workflows/build-test-publish.yml

```diff
@@ -15,7 +15,8 @@ jobs:
             - name: Checkout DFG-Viewer repository
               uses: actions/checkout@v4
               with:
-                  repository: slub/dfg-viewer
+                  repository: ${{ env.GIT_OWNER }}/${{ env.GIT_REPO }}
```

Comment:
Hm, this does not work: neither env.GIT_OWNER nor env.GIT_REPO are defined, so the action fails because an invalid repository '/' is choosen. How could this work previously?

Should `${{ github.repository_owner }}` and `${{ github.event.repository.name }}` be used?

Reply:
Thx for the hint. Please use default env variables
https://docs.github.com/en/actions/writing-workflows/choosing-what-your-workflow-does/store-information-in-variables#default-environment-variables
I think $GITHUB_REPOSITORY would be sufficient without curly brackets

Reply:
I now used context variables because the default env variables without curly brackets did not work.